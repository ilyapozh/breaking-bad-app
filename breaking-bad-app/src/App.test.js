// import { render, screen } from '@testing-library/react';
// import App from './App';

// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });
// import App from "./App";
const {fetchEpisodes} = require('./App');

it("Checks fetch episodes for return Promise", () => {
    expect(fetchEpisodes).toBe(Promise)
})