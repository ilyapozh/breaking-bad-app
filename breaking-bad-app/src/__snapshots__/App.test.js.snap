// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col"
  >
    <h1
      class="w-fit text-6xl self-center border-b-2 m-8"
      data-testid="title"
    >
      Breaking bad episodes
    </h1>
  </div>
</DocumentFragment>
`;
